tofu/geom/utils.py

```diff
@@ -819,51 +833,83 @@ def _create_config_testcase(
-                           + "    Matching: "+"\n              ".join(ff))
-                    raise Exception(msg)
+                if len(ff) == 0:
+                    msg = (
+                        "No matching files in {} ".format(path)
+                        + "for ['{}', '{}', '{}']".format(cc, ssbis, Expbis)
+                    )
+                    dfail[k0] = msg
+                    continue
+                elif len(ff) > 1:
+                    msg = (
+                        "Many ({}) matching files in {} ".format(len(ff), path)
+                        + "for ['{}', '{}', '{}']".format(cc, ssbis, Expbis)
+                    )
+                    dfail[k0] = msg
+                    continue
 
             pfe = os.path.join(path, ff[0])
             try:
-                obj = eval('_core.'+cc).from_txt(pfe, Name=ss, Type='Tor',
-                                                 Exp=dconfig[config]['Exp'],
-                                                 returnas=returnas)
+                obj = eval('_core.'+cc).from_txt(
+                    pfe, Name=ss, Type='Tor',
+                    Exp=dconfig[config]['Exp'],
+                    returnas=returnas,
+                )
                 if returnas not in ['object', object]:
-                    obj = ((ss, {'Poly': obj[0],
-                                 'pos': obj[1], 'extent': obj[2]}),)
+                    obj = ((
+                        ss, {'Poly': obj[0], 'pos': obj[1], 'extent': obj[2]}
+                    ),)
                 lS.append(obj)
             except Exception as err:
-                msg = "Could not be loaded: {}".format(ff[0])
-                warnings.warn(msg)
+                msg = "Error loading: {}".format(str(err))
+                dfail[k0] = msg
+                continue
+
+    if len(dfail) > 0:
+        lstr = ['\t- {}: {}'.format(k0, v0) for k0, v0 in dfail.items()]
+        msg = (
+            "The following structures could not be loaded:\n"
+            + "\n".join(lstr)
+        )
+        if strict is True or len(lS) == 0:
+            raise Exception(msg)
+        else:
+            warnings.warn(msg)
+
     if returnas == 'dict':
         conf = dict([tt for tt in lS])
     else:
         conf = _core.Config(Name=config, Exp=dconfig[config]['Exp'], lStruct=lS)
     return conf
 
 
-def create_config(case=None, Exp='Dummy', Type='Tor',
-                  Lim=None, Bump_posextent=[np.pi/4., np.pi/4],
-                  R=None, r=None, elong=None, Dshape=None,
-                  divlow=None, divup=None, nP=None,
-                  returnas=None, SavePath='./', path=_path_testcases):
+def create_config(
+    case=None, Exp='Dummy', Type='Tor',
+    Lim=None, Bump_posextent=[np.pi/4., np.pi/4],
+    R=None, r=None, elong=None, Dshape=None,
+    divlow=None, divup=None, nP=None,
+    returnas=None, strict=None,
+    SavePath='./', path=_path_testcases,
+):
```

Comment:
Could you update the doc below? since this function is directly used by the users

Reply:
Done   :-)

Reply:
P.S.: @lasofivec  I would like to release 1.4.11 today or tomorrow, shall I wait for a PR on your side?

Reply:
yes, coming in a couple of minutes

Reply:
OK, I'll do it at the end of the day, probably around 16h-17h, so take your time to polish the PR :-)